django/db/migrations/autodetector.py

```diff
@@ -17,6 +18,22 @@
     resolve_relation,
 )
 
+OperationDependency = namedtuple(
+    "OperationDependency", "app_label model_name field_name type"
+)
+
+
+class DependencyType(Enum):
+    CREATE = 0
+    REMOVE = 1
+    ALTER = 2
+    ORDER_WRT_UNSET = 3
+    FOO_TOGETHER_CHANGE = 4
```

Comment:
Could it be possible to use more explicit names without abbreviations?

Reply:
what about `ORDER_WRT_UNSET`? can we use a more explicit name?

Reply:
I also think that "foo" isn't all that great here. It was clearly a way of trying to express both "index_together" and "unique_together", but I'm not sure why we didn't just end up with "index_or_unique_together".

I was wondering whether we'd be better with:

```python
    REMOVE_ORDER_WITH_RESPECT_TO = 3
    ALTER_INDEX_OR_UNIQUE_TOGETHER = 4
```

It feels as though these would sync up better with the operation classes: `AlterIndexTogether`, `AlterUniqueTogether`, and `AlterOrderWithRespectTo`. (Also just use "remove" instead of "unset" and "alter" consistently instead of "change".)

Reply:
> It feels as though these would sync up better with the operation classes: AlterIndexTogether, AlterUniqueTogether, and AlterOrderWithRespectTo. (Also just use "remove" instead of "unset" and "alter" consistently instead of "change".)

But would not sync with autodetector methods and variables.

Reply:
I changed to `REMOVE_` and `ALTER_` prefixes.

Reply:
I guess I don't mind "foo" as we've used it in places like get_foo_display() 🤷‍♂️  It's a little weird but seems somewhat conventional.

Reply:
I'm not going to lose sleep over it 😄 With the change to the suggested prefixes the "foo" ain't so bad.